server/Authentication/authentication.ts

```diff
@@ -0,0 +1,45 @@
+import express from 'express'
+import dotenv from 'dotenv';
+import bcrypt from 'bcrypt';
+import { validateUserCredentials } from '../middlewares/registrationMiddleware'
+import client from '../database/database'
+
+const app = express();
+app.use(express.json());
+dotenv.config();
+
+//Test porpouses only
+app.get('/registration', (req: express.Request, res: express.Response) => {
+    const test = client.query('SELECT * FROM users');
```

Comment:
This query returns all users. But better filter some information to get fewer data.